gc/base/GCExtensionsBase.hpp

```diff
@@ -854,13 +854,13 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	bool pretouchHeapOnExpand; /**< True to pretouch memory during initial heap inflation or heap expansion */
 
-	uintptr_t decommitMinimumFree;   /**< percentage of free heap to be retained as committed, default=0 for gencon, complete tenture free memory will be decommitted */
-
+	uintptr_t decommitMinimumFree; /**< percentage of free heap to be retained as committed, default=0 for gencon, complete tenture free memory will be decommitted */
+	float gcOnExplicitCompactThreshold; /**< Enables compaction when fragmented memory and dark matter exceed this limit. The larger this number, the more memory can be fragmented before compact is triggered **/
```

Comment:
A more appropriate name is pageFragmentationCompactThreshold. And while there, I'd update the comment too: ...page-fragmented memory...

And now it become more generic, it can sit next to another threshold: darkMatterCompactThreshold.